src/scenes/AcceleratorRouter/DeliverableCellRenderer.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Badge } from '@terraware/web-components';
+import CellRenderer, { TableRowType } from 'src/components/common/table/TableCellRenderer';
+import { RendererProps } from 'src/components/common/table/types';
+
+export default function DeliverableCellRenderer(props: RendererProps<TableRowType>): JSX.Element {
+  const { column, row, index } = props;
+
+  if (column.key === 'status') {
+    // TODO convert BE label value to locale specific string
```

Comment:
there's a categoryLabel func in types/Deliverables.. can add something similar for statusLabel